pkg/url/url.go

```diff
@@ -159,7 +160,7 @@ func Create(client *occlient.Client, kClient *kclient.Client, parameters CreateP
 		}
 		ingress := generator.GetIngress(ingressParam)
 		// Pass in the namespace name, link to the service (componentName) and labels to create a ingress
-		i, err := kClient.CreateIngress(*ingress)
+		i, err := kClient.CreateExtensionV1Beta1Ingress(*ingress)
```

Comment:
Same as `DeleteIngress` vs. `DeleteExtensionV1Beta1Ingress`.

Reply:
Same answer, part 2 of pr will do this